src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-param-reassign */
+import React, { useEffect, useState } from 'react';
+import { Loader } from './Loader/Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[] | []>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  const noErrorAndLoad = !isError && !isLoading;
+
+  const normalizePeople = people.map(person => {
+    const father = people.find(parent => person.fatherName === parent.name);
+    const mother = people.find(parent => person.motherName === parent.name);
```

Comment:
try to optimize this search to make only one `find`. You can make both condition inside `find`